src/components/FeedbackSurvey.tsx

```diff
@@ -0,0 +1,86 @@
+import {CONST as COMMON_CONST} from 'expensify-common/lib/CONST';
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {TranslationPaths} from '@src/languages/types';
+import FixedFooter from './FixedFooter';
+import FormAlertWithSubmitButton from './FormAlertWithSubmitButton';
+import SingleOptionSelector from './SingleOptionSelector';
+import Text from './Text';
+
+type FeedbackSurveyProps = {
+    /** Title of the survey */
+    title: string;
+
+    /** Description of the survey */
+    description: string;
+
+    /** Callback to be called when the survey is submitted */
+    onSubmit: (reason: Option) => void;
+};
+
+type Option = {
+    key: string;
+    label: TranslationPaths;
+};
+
+const OPTIONS: Option[] = [
```

Comment:
If you created this component under `src/components/` with the intention to be reusable in other places of the app, it would make more sense to pass the options as a prop instead of hardcoding them here. If was not the intention to be reusable, could we place the component inside `pages/settings/Subscription/` instead?

Reply:
I had same thought when I've started this task but it turns out that this FeedbackSurvey component will be reusable and will always have the same questions thus options are not passed via props

Reply:
@MrMuzyk  But will it be reusable between the Subscription's pages / components, or could be part of a page / component of any other feature?

1. If it's going to be reusable only between Subscription scope, I would suggest moving it to `pages/settings/Subscription/`.
2. If it's going to be reusable between any App feature or score, I would suggest at least renaming to `SubscriptionFeedbackSurvey` since the questions are related to subscription only.

WDYT?

Reply:
It will be reusable across the app with same options. Maybe CONST naming in expenisfy-common is a bit unfortunate so I will create a CONST variable on our end that has a name that fits reusable component a bit better